utils/src/search_xattrs.c

```diff
@@ -96,14 +96,11 @@ static int parse_opt(int key, char *arg, struct argp_state *state)
 			argp_error(state, "more than one name argument given");
 
 		args->name = strdup_or_error(state, arg);
-		if (strncmp(args->name, SCOUTFS_XATTR_PREFIX, strlen(SCOUTFS_XATTR_PREFIX)))
-			argp_error(state, "xattr must start with \""
-				     SCOUTFS_XATTR_PREFIX"\"");
 		break;
 	case ARGP_KEY_FINI:
 		if (!args->name) {
 			argp_error(state, "must provide xattr name starting with \""
```

Comment:
"containing .srch. scoutfs tag"?    Haven't really documented this stuff for users so I don't think we have great language and terms yet.  Just don't want to give the impression that it has to start with scoutfs.srch.